### PR TITLE
Fix: update dependency-check-maven version to 5.2.1 and all needed li…

### PR DESCRIPTION
**Pull request by JSLair**
…braries

- Change versions
-- cxf 3.2.0 -> 3.3.3
-- bouncycastle (in truezip innner dependency) 1.52 -> 1.62
-- common-compress 1.14 -> 1.18
-- junrar 0.7 -> 4.0.0

- Prevent System scope analysis
- Fix some maven-dependency-plugin errors

### CODE REVIEW

File: droid-binary/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <artifactId>droid-parent</artifactId>
         <groupId>uk.gov.nationalarchives</groupId>
-        <version>6.5-SNAPSHOT</version>
+        <version>6.5-VITAM</version>
```
Comment (adamretter) [inline droid-binary/pom.xml]: Can you remove these version changes please?

File: droid-command-line/pom.xml [hunk 2]
```diff
@@ -46,10 +46,11 @@
                     <dependency>
                         <groupId>uk.gov.nationalarchives</groupId>
                         <artifactId>droid-build-tools</artifactId>
-                        <version>6.5-SNAPSHOT</version>
+                        <version>6.5-VITAM</version>
                     </dependency>
                 </dependencies>
             </plugin>
+<!--
```
Comment (adamretter) [inline droid-command-line/pom.xml]: Why is the `maven-dependency-plugin` commented out now?

File: droid-swing-ui/src/test/java/uk/gov/nationalarchives/droid/gui/filechooser/ResourceFileChooserTest.java [hunk 3]
```diff
@@ -1,70 +1,71 @@
...
+        //UIManager.setLookAndFeel("com.sun.java.swing.plaf.motif.MotifLookAndFeel");
+        
+        fc = new ResourceFileChooser();
+    }
+    
+    @Test
+    public void testFileChooserSetup() {
+        
+        assertTrue(fc.isAcceptAllFileFilterUsed());
+        JFileChooser defaultFileChooser=new JFileChooser("");
```
Comment (adamretter) [inline droid-swing-ui/src/test/java/uk/gov/nationalarchives/droid/gui/filechooser/ResourceFileChooserTest.java]: Why is this change needed?

File: droid-binary/pom.xml (hunk 1 above)
Comment (JSLair) [reply droid-binary/pom.xml]: Yes for sure...

File: droid-swing-ui/src/test/java/uk/gov/nationalarchives/droid/gui/filechooser/ResourceFileChooserTest.java (hunk 3 above)
Comment (JSLair) [reply droid-swing-ui/src/test/java/uk/gov/nationalarchives/droid/gui/filechooser/ResourceFileChooserTest.java]: This is the heart of this merge request.
When you are on non-english environment the good answer is not "All files", for example it's "Tous les fichiers". This modification is langage independant...

File: droid-command-line/pom.xml (hunk 2 above)
Comment (JSLair) [reply droid-command-line/pom.xml]: Becaused it's managed at the general level droid-parent

File: droid-swing-ui/src/test/java/uk/gov/nationalarchives/droid/gui/filechooser/ResourceFileChooserTest.java (hunk 3 above)
Comment (JSLair) [reply droid-swing-ui/src/test/java/uk/gov/nationalarchives/droid/gui/filechooser/ResourceFileChooserTest.java]: It's the already merged modification #234

**Comment by adamretter** [comment]
@JSLair I wanted to get Travis working again before I merged any more external PRs. Can you rebase this one now please?

**Comment by JSLair** [comment]
I have to do that on another branch...
So I open a new Pull Request
